### ISSUE TITLE
color bar

### ISSUE THREAD

**Issue by cmyuan6** [cmyuan6/CLIM680#1]
One issue I have encountered with my subplots is that the trend in precipitation during different months is not particularly clear since I am using a single large color bar for all 12 months. With the minimum value being 0 and the maximum being over 267, the overall gradation in the color mapping does not effectively represent the variation in precipitation levels across the months. I have experimented with various color maps, including Blues, Viridis, RdBu_r, and so on, but none have succeeded in showing the gradients without compromising the clarity of the graphs. One potential improvement could be to have three to four separate color bars corresponding to each season so that months with similar precipitation patterns can be displayed on the same scale. However, I think that using multiple color bars could make the figure look cluttered.

**Comment by luisortizuriarte** [comment]
You are correct that the current choice of colormap does not appear to be a good fit for the monthly precipitation data. One option you can try is to use a different normalization. By default, matplotlib maps data values to colors linearly. However, there are other options. You can try to specify specific boundaries or use one of the pre-fit functions (power law, log). Another option is to mask out very low values or even to use the 5th and 95th percentiles of precip to create your clevs rather than min/max.

https://matplotlib.org/stable/users/explain/colors/colormapnorms.html

**Comment by cmyuan6** [comment]
I have used the 5th and 95th percentiles of precip to create my clevs to reflect better color schemes for the map.

**Comment by luisortizuriarte** [comment]
Looks good!
